### ISSUE TITLE
`from` breaks when a struct member is feature gated

### ISSUE THREAD

**Issue by kaspar030** [roignpar/optfield#13]
Generation of `from()` breaks when some struct fields are feature gated but not used.

E.g.:

```
❯ cat -p src/main.rs 
use optfield::optfield;

#[optfield(OptFoo, attrs, field_attrs, from)]
struct Foo {
    a: String,
    #[cfg(feature = "foo")]
    b: String,
}

fn main() {
    println!("Hello, world!");
}
```

... expands to this:

```
#![feature(prelude_import)]                                                    
#[prelude_import]    
use std::prelude::rust_2021::*;
#[macro_use]                        
extern crate std;      
use optfield::optfield;
struct Foo {
    a: String,      
}                                      
struct OptFoo {
    a: Option<String>,                                                                                                                                         
}                                    
impl From<Foo> for OptFoo {                                                                                                                                    
    fn from(item: Foo) -> OptFoo {
        OptFoo {
            a: Some(item.a),                                                   
            b: Some(item.b),
        }     
    }                      
}             
fn main() {
    {
        ::std::io::_print(format_args!("Hello, world!\n"));
    };                        
}
```

... which, unless the feature foo is enabled, fails compiling:

```
❯ cargo build
   Compiling optfield_cfgtest v0.1.0 (/home/kaspar/tmp/optfield_cfgtest)
error[E0560]: struct `OptFoo` has no field named `b`
 --> src/main.rs:7:5
  |
7 |     b: String,
  |     ^ `OptFoo` does not have this field
  |
  = note: all struct fields are already assigned

error[E0609]: no field `b` on type `Foo`
 --> src/main.rs:7:5
  |
7 |     b: String,
  |     ^ unknown field
  |
help: a field with a similar name exists
  |
7 |     a: String,
  |     ~


```

**Comment by kaspar030** [comment]
Can any '#[cfg(...)] be copied over to the corresponding field setter in the `from()` implementation?

**Comment by roignpar** [comment]
@kaspar030 this was a bug. Thank you for the issue and PR. Fixed and released version `0.4.0`.
